ts-defs/index.d.ts

```diff
@@ -1534,6 +1693,25 @@ interface FixtureFn {
      * Skips execution of all tests, except whose that are in this fixture.
      */
     only: this;
+    /**
+     * Specifies the additional information for all tests in the fixture that can be used in reports
```

Comment:
Specifies the additional information for all tests in the fixture. This information can be used in reports.